doc/source/theory/multiphysics/heat_transfer/heat_equation.rst

```diff
@@ -2,4 +2,50 @@
 Heat Transfer Equations
 ================================
 
+In Lethe, it is possible to solve the heat equation in various loading conditions. The main equation is derived from the energy equation in incompressible fluids. Assuming a constant heat capacity :math:`c_p` and dynamic viscosity :math:`\mu`, the equation takes the following form: 
```

Comment:
```suggestion
In Lethe, it is possible to solve the heat equation in various loading conditions. The main equation is derived from the energy equation in incompressible fluids. Assuming a constant heat capacity :math:`c_p` and dynamic viscosity :math:`\mu`, the equation takes the following form: 
```

heat transfer or energy equation. it's not really the heat equation per se because it describes energy conservation.

 Can you look at the physical properties sections to check if we use $C_p$ or $c_p$?